### PR TITLE
Redesign: Generic panel

### LINKED ISSUE #6211: Redesign: Generic panel
## Feature Description

Update design for a generic panel based on [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3360%3A63235).

- [ ] Update arrow icon using design system
- [ ] Update title and typography using design system
- [ ] Update focus style

| Current design | New design |
|-|-|
| Screen Shot 2021-02-04 at 23 57 27 | Screen Shot 2021-02-04 at 23 56 56 |

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by miina**
## User-facing changes
| Before | After |
|--------|------|
| Screenshot 2021-02-25 at 14 50 35  | Screenshot 2021-02-25 at 14 49 44 |

### QA
This can be QA-d per-panel.

- [x] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1.

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [ ] I have tested this code to the best of my abilities
- [ ] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testiing.md))
- [ ] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [ ] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [ ] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #6211

### CODE REVIEW

**Comment by miina** [comment]
@barklund IDK right now where the extra white outline comes from in focused state, however, it feels like we should address this together in the whole editor instead of panel by panel -- we shouldn't have such inconsistencies when releasing anyway.
Perhaps a general issue for fixing the focus considering the keyboard only rule, too (also for inputs, see https://github.com/google/web-stories-wp/pull/6489#discussion_r582764094 as well) would make sense.

File: assets/src/edit-story/components/panels/panel/shared/title.js [hunk 1]
```diff
@@ -62,33 +51,35 @@ const Header = styled.h2.attrs({ role: 'button' })`
...
-  margin: 0;
-  font-weight: 500;
-  font-size: 14px;
-  line-height: 19px;
-  width: 100%;
+const Heading = styled(Headline)`
+  color: ${({ theme }) => theme.colors.fg.secondary};
+  line-height: 32px;
+  font-weight: 700;
+  font-size: 16px;
```
Comment (barklund) [inline assets/src/edit-story/components/panels/panel/shared/title.js]: Can you add the proper headline size to the typography styles in the design system as per @ivayloklissarov's [comment](https://xwp.slack.com/archives/C01ESBB8ZN1/p1614274481039000?thread_ts=1614256943.035200&cid=C01ESBB8ZN1)?

File: assets/src/edit-story/components/panels/panel/shared/title.js [hunk 2]
```diff
@@ -20,35 +20,24 @@
...
-}
-
 const Header = styled.h2.attrs({ role: 'button' })`
-  background-color: ${({ isPrimary, isSecondary, theme }) =>
-    getBackgroundColor(isPrimary, isSecondary, theme)};
-  border: 0 solid
-    ${({ theme }) => rgba(theme.DEPRECATED_THEME.colors.fg.gray16, 0.6)};
-  border-top-width: ${({ isPrimary, isSecondary }) =>
-    isPrimary || isSecondary ? 0 : '1px'};
+  background-color: ${({ theme }) => theme.colors.bg.secondary};
```
Comment (barklund) [inline assets/src/edit-story/components/panels/panel/shared/title.js]: Does it even need a background color? Can't it just be transparent here?

**Review by barklund — APPROVED** [review]
Looks good - just need to fix the typography (move to design system) and then we can follow-up to fix the focus stuff.

File: assets/src/edit-story/components/panels/panel/shared/title.js (hunk 1 above)
Comment (miina) [reply assets/src/edit-story/components/panels/panel/shared/title.js]: Done in 720c268

**Pull request merged**